invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -256,18 +258,124 @@ class CommunityProfileForm extends Component {
       },
     });
 
+    /**
+     * Deserializes a funding record (e.g. funder or award)
+     *
+     * @param {object} fund
+     *
+     * @returns {object} an object containing the deserialized record
+     */
+     const deserializeFunding = ((fund) => {
```

Comment:
This is exactly the same logic to deserialise a funding record that was used in  https://github.com/inveniosoftware/react-invenio-deposit/blob/aefebaa2156c752ee10829d69c85b3ab22a15eeb/src/lib/fields/VocabularyField.js#L42-L44

Reply:
Can you just import the field and use the deserialize method? Maybe it's not exported but we could export the field eventually....